### PR TITLE
New package: breeze-cursor-themes-1.0

### PR DESCRIPTION
**Pull request by noblehelm**

### CODE REVIEW

File: srcpkgs/breeze-cursor-themes/template [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
...g/web/20150803151134/https://share.kde.org/index.php/s/1525897da49fbebacc28e5b6b1836382/download>breeze_default.zip http://web.archive.org/web/20150803065118/https://share.kde.org/index.php/s/3e86f7cd39dd9db909da9c7ba5b3de5d/download>breeze_contrast.zip http://web.archive.org/web/20150803065147/https://share.kde.org/index.php/s/66c54f4c02790f979b17c131a07f1b69/download>breeze_hacked.zip http://web.archive.org/web/20150803065121/https://share.kde.org/index.php/s/6524eea49b2f0f30be31e98e9fe6b731/download>breeze_obsidian.zip http://web.archive.org/web/20150803065108/https://share.kde.org/index.php/s/0961e0d81ccf38e12950e1fc435450a4/download>breeze_snow.zip http://web.archive.org/web/20150803065113/https://share.kde.org/index.php/s/3f558bd504f8365141f2ccaf0426c237/download>breeze_amber.zip http://web.archive.org/web/20150803065112/https://share.kde.org/index.php/s/7e177568d01d2016eecbcf246e38f061/download>breeze_purple.zip http://web.archive.org/web/20150803065143/https://share.kde.org/index.php/s/cf72875d375d5a7746c921dc2d8d2499/download>breeze_blue.zip http://web.archive.org/web/20150803065124/https://share.kde.org/index.php/s/67ff3ae466f4670a8edb25d2d67f3c7e/download>breeze_red.zip"
```
Comment (abenson) [inline srcpkgs/breeze-cursor-themes/template]: Break these out onto multiple lines just like the checksums.

Also, archive.org?

**Comment by cr6git** [comment]
@noblehelm I took the liberty to modify your template, to fix and to improve. See [b4d26ce](https://github.com/cr6git/void-packages/commit/b4d26cedf4379818c120f5f0eda93495ff8de160) :gift:

**Comment by noblehelm** [comment]
@cr6git Yes, that is perfect, can you please make a new PR for this package? And change the maintainer to your name, which will be best. I'll close this PR.

**Comment by cr6git** [comment]
@noblehelm it's done! [eb82cfd](https://github.com/voidlinux/void-packages/commit/eb82cfd999e3b4ec25733bbea7629b494545b413) :tada:

**Pull request closed without merging**